_posts/guilty-feeling.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "The guilt that comes with flexible working hours"
+author: Mauro Servienti
+synopsis: "I've been working remotely for Particular Software for the last ten years. To this day, I still feel guilty for taking time off during regular working hours, even though we don't have working hours at all. Why is that?"
+header_image: /img/posts/guilty-feeling/header.jpg
+tags:
+- remote-work
+- time-management
+---
+
+I've been working remotely for [Particular Software](https://www.particular.net) for the last ten years. To this day, I still feel guilty for taking time off during regular working hours, even though we don't have working hours at all.
+
+Society is built around working hours. Schools start before we go to work, and often, schoolday ends after we finish working. Retail and grocery shops are still open when we stop working, and so on.
```

Comment:
```suggestion
Society is built around working hours. Schools start before we go to work, and often, the school day ends after we finish working. Retail and grocery shops are still open when we stop working, and so on.
```

Is this true in Europe? In North America, the standard school day starts around 8am and finishes around 3pm. I.e. it might start before work but it usually ends before the end of the workday.

Reply:
```suggestion
Society is built around working hours. Schools start before we go to work, and often, the school day ends after we finish work, for example, in Italy we have pre and post-school making it so that primary school ends at 6 PM. Retail and grocery shops are still open when we stop working, and so on.
```